test/outtest2.F90

```diff
@@ -102,7 +104,14 @@ program outtest2
   ! Confirm the "missing" value is still the same value that was set previously via the call to setxmiss.
   IF ( nint(xmiss) .ne. nint(getbmiss()) ) stop 3
 
-  call writsb ( 11 )
+  ! Test cnved4 to cover im8b=.true. case
+  call writsa ( 11, mxbfmg, mgbf, lmgbf )
+  mgbf2 = mgbf
+  call cnved4(mgbf,mxbfmg,mgbf2)
```

Comment:
Why do you need to copy mgbf into mgbf2 in line 109?  The call to cnved4 in line 110 will already do that itself, won't it?

Reply:
Just removed the `mgbf2 = mgbf`